src/main/kotlin/app/revanced/patches/youtube/layout/sponsorblock/bytecode/fingerprints/ShortsPlayerConstructorFingerprint.kt

```diff
@@ -33,6 +36,16 @@ object ShortsPlayerConstructorFingerprint : MethodFingerprint(
         Opcode.IPUT_OBJECT,
         Opcode.NEW_INSTANCE,
         Opcode.INVOKE_DIRECT,
-        Opcode.IPUT_OBJECT
-    )
+        Opcode.IPUT_OBJECT,
+        Opcode.IPUT_OBJECT,
+        Opcode.CONST_4
+    ),
+    customFingerprint = { methodDef ->
+        methodDef.implementation?.instructions?.any { instruction ->
+            (instruction as? WideLiteralInstruction)?.wideLiteral ==
+                    ResourceIdMappingProviderResourcePatch.resourceMappings.single {
+                        it.type == "id" && it.name == "reel_persistent_edu_button_group"
+                    }.id
```

Comment:
Keep in mind this will call `.single` every time the `customFingerprint´ function is executed. Please move it outside of the function.